tools/crm_simulate.c

```diff
@@ -73,6 +76,274 @@ extern gboolean bringing_nodes_online;
 	}					\
     } while(0)
 
+#define INDENT "                                   "
+
+static gboolean
+in_place_cb(const gchar *option_name, const gchar *optarg, gpointer data, GError **error) {
+    options.store = TRUE;
+    options.process = TRUE;
+    options.simulate = TRUE;
+    return TRUE;
+}
+
+static gboolean
+live_check_cb(const gchar *option_name, const gchar *optarg, gpointer data, GError **error) {
+    if (options.xml_file) {
+        free(options.xml_file);
+    }
+
+    options.xml_file = NULL;
+    return TRUE;
+}
+
+static gboolean
+node_down_cb(const gchar *option_name, const gchar *optarg, gpointer data, GError **error) {
+    options.modified++;
+    options.node_down = g_list_append(options.node_down, (gchar *) optarg);
+    return TRUE;
+}
+
+static gboolean
+node_fail_cb(const gchar *option_name, const gchar *optarg, gpointer data, GError **error) {
+    options.modified++;
+    options.node_fail = g_list_append(options.node_fail, (gchar *) optarg);
+    return TRUE;
+}
+
+static gboolean
+node_up_cb(const gchar *option_name, const gchar *optarg, gpointer data, GError **error) {
+    options.modified++;
+    bringing_nodes_online = TRUE;
```

Comment:
There's another thing I never noticed before. Probably another good candidate for pe_flag_*

Reply:
I could add that to this PR.  `bringing_nodes_online` is only used in two places.

Reply:
Let's leave it for later, I'm not sure we have data_set readily available where it's needed